tmt-app/src/Components/Quiz.jsx

```diff
@@ -51,36 +52,41 @@ function Text({ step }) {
   const content = textContent[step] || textContent["classes"];
 
   return (
-    <Grid
-      container
-      direction="row"
-      display="flex"
-      justifyContent="space-between"
-      alignItems="center"
-      paddingX={5}
-      sx={{
-        "@media (max-width: 1000px)": {
-          justifyContent: "center",
-          textAlign: "center",
-        },
-      }}
-    >
-      <Grid item>
-        <Grid container direction="column" display="flex">
-          <Typography variant="h3" mb={3}>
-            {content.title1}
-          </Typography>
-          <Typography variant="h3">{content.title2}</Typography>
-          <Typography variant="h5" color={CustomTheme.palette.ThaiTea.main}>
-            {content.subtitle}
-          </Typography>
-          <div style={{ height: 40 }} />
+    <>
+      <Box sx={{ my: "3rem" }}>
+        <QuizStepper />
+      </Box>
```

Comment:
If we're going to wrap every stepper in 3rem of y margin, could we just put this in the stepper component?